tests/test_utils.py

```diff
@@ -20,12 +22,28 @@ def setup_method(self):
         self.df_tessellation = gpd.read_file(test_file_path, layer="tessellation")
         self.df_streets = gpd.read_file(test_file_path, layer="streets")
         self.df_buildings["height"] = np.linspace(10.0, 30.0, 144)
+        self.df_points = gpd.GeoDataFrame(
+            data=None, geometry=[Point(0, 0), Point(1, 1)]
+        )
+        self.df_points_and_linestring = gpd.GeoDataFrame(
+            data=None,
+            geometry=[Point(0, 0), Point(1, 1), LineString([(1, 0), (0, 0), (1, 1)])],
+        )
 
     def test_dataset_missing(self):
         with pytest.raises(ValueError):
             mm.datasets.get_path("sffgkt")
 
     def test_gdf_to_nx(self):
+
+        # nx = mm.gdf_to_nx(self.df_points)
+        with pytest.warns(RuntimeWarning):
```

Comment:
```suggestion
        with pytest.warns(RuntimeWarning, match="The given network does not contain any LineString."):
```

This is a more precise check that we raise the exact warning we expect.